test/ui-test/testSuites/suite_status/shared/steps/chatSteps.py

```diff
@@ -73,6 +73,15 @@ def step(context, message):
 def step(context, message_index, message):
     _statusChat.reply_to_message_at_index(message_index, message)
     _statusChat.verify_last_message_sent(message)
+    
+@When("the user can edit the message at index |any| and change text to \"|any|\"" )
+def step(context, message_index, message):
+    _statusChat.edit_message_at_index(message_index, message)
+    _statusChat.verify_last_message_sent(message)
+
+@Then("the user can read the message marked as \"|any|\"")
```

Comment:
Since this only checks the last message, let's make it clearer
```suggestion
@Then("the last message is marked as \"|any|\"")
```

Reply:
Otherwise, instead of adding a new step that most likely only works for this specific use case, you can reuse `@Then("the message |any| is displayed in the last message")` as 
```
Then the message (Edited) is displayed in the last message
```